packages/side-nav/src/SideNav.spec.tsx

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, cleanup } from '@testing-library/react';
+import { MenuItemProps } from '@leafygreen-ui/menu';
+import { SideNav, SideNavGroup, SideNavItem } from './index';
+
+type renderedElement = HTMLElement | null | undefined;
```

Comment:
Because you are implementing this type as optional, undefined is already inferred for you so you shouldn't need tot explicitly define here